tools/plugins/com.liferay.ide.upgrade.tasks.core/src/com/liferay/ide/upgrade/tasks/core/internal/prerequisite/CheckInstalledJDKsAction.java

```diff
@@ -20,7 +20,6 @@
 
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
-
```

Comment:
Hey @christopherbryanboyd it will cause sf failed if we remove this line

Reply:
Hi sorry about that @jtydhr88 , I have submitted a fix